routes/web.php

```diff
@@ -39,5 +39,6 @@
 Route::get('build', 'LoadoutsController@build');
 Route::get('build/{loadoutId}', 'LoadoutsController@build')->name('build.show');
 Route::view('/privacy-policy', 'privacy-policy.index');
+Route::get('asv/{gun}/{combo}', 'BuildMetricController@index')->name('asv.index')->where('gun', '[0-9]+');
```

Comment:
Can we change this to a `show` actually? I think that is more proper in this case since it is a specific build combo we are looking at.

Reply:
Route::show doesn't exist. What do you mean?

Reply:
```suggestion
Route::get('asv/{gun}/{combo}', 'BuildMetricController@show')->name('asv.show')->where('gun', '[0-9]+');
```

Reply:
I've done this, can you just explain why you would do this vs. using index as the method name? 

From my perspective I don't see a clear difference, but it sounds like this is one of those convention things that I am just not as familiar with. Just so I know why moving forward.

Thanks

Reply:
Yeah, it's just a convention. In "REST" design `index` is when you get a list of something (like `/products`). `show` is when you get 1 of a thing (like `/products/1`). In this case, we are showing a single combo, not a list, so the convention says we should make it a `show`.

Reply:
💯 Preciate it, thank you sir